lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -162,6 +168,12 @@ public Map<String, Object> handleRequest(JourneyRequest journeyRequest, Context
         }
     }
 
+    private void saveCoiJourneyType(String journeyEvent, IpvSessionItem ipvSessionItem) {
+        CoiSubjourneyTypes coiJourneyType = CoiSubjourneyTypes.fromString(journeyEvent);
+
+        ipvSessionItem.setCoiSubjourneyTypes(coiJourneyType);
```

Comment:
Could this be inlined? If so, you could then drop this method and call `setCoiSubjourneyTypes()` in the `if` statement.